ccmbioinfo/lfs
# CARDS-1856: Fix java build warnings

[CARDS-1856](https://phenotips.atlassian.net/browse/CARDS-1856)

To test:
- Does the build work correctly?
- Does CSV data import work?
- Does indexing NCIT with `curl -v -u admin:admin -X POST http://localhost:8080/Vocabularies -d source=ncit-flat -d version=22.06d` work? Also try to make a question use it and check that using it is fine.
